dags/example_dataproc.py

```diff
@@ -0,0 +1,51 @@
+from airflow import DAG
+from airflow.utils import trigger_rule
+from datetime import datetime, timedelta
+
+from utils.dataproc import moz_dataproc_runner
+
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+from airflow.operators.subdag_operator import SubDagOperator
+
+default_args = {
+    'owner': 'hwoo@mozilla.com',
+    'depends_on_past': False,
+    'start_date': datetime(2099, 7, 30),
+    'email': ['telemetry-alerts@mozilla.com', 'hwoo@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 2,
+    'retry_delay': timedelta(minutes=30),
+    # Change schedule_interval to @daily, @weekly, or cron syntax to schedule the dag to run
+    'schedule_interval': None,
+}
+
+dag = DAG('example_dataproc', default_args=default_args)
+
+# Unsalted cluster name so subsequent runs fail if the cluster name exists
+cluster_name = 'test-dataproc-cluster-hwoo'
+
+# defined in Airflow's UI -> Admin -> Connections
+gcp_conn_id = 'google_cloud_airflow_dataproc'
+connection = GoogleCloudBaseHook(gcp_conn_id=gcp_conn_id)
+
+# moz_dataproc_runner creates a spark dataproc cluster, runs a script uri on it similar
+# to our aws custom emrsparkoperator (script-runner.jar -> airflow_gcp.sh -> script uri),
+# and then deletes the dataproc cluster.
+# Ipynb script uris are deprecated.
+
+run_dataproc_spark = SubDagOperator(
+    task_id='run_dataproc_dag',
+    email=["telemetry-alerts@mozilla.com", "hwoo@mozilla.com"],
```

Comment:
Do you need to specify email here? You could pass in`default_args['email']` since it is already defined.

Reply:
Removing. Thanks